src/app/thumbnails.h

```diff
@@ -0,0 +1,56 @@
+// Aseprite
+// Copyright (C) 2016  Carlo Caputo
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 2 as
+// published by the Free Software Foundation.
+
+#ifndef APP_THUMBNAILS_H_INCLUDED
+#define APP_THUMBNAILS_H_INCLUDED
+#pragma once
+
+#include "base/shared_ptr.h"
+#include "base/connection.h"
+#include "she/surface.h"
+#include "doc/object_id.h"
+#include "doc/object.h"
+#include "doc/frame.h"
+
+#include <map>
+#include <set>
+#include <utility>
+
+namespace doc {
+  class Cel;
+  class Image;
+  class Document;
+}
+
+namespace app {
+  class Document;
+
+  namespace thumb {
+
+    typedef std::pair< int, int > Dimension;
```

Comment:
Could this `Dimension` type be replaced with a `gfx::Size`? Because it looks like it's acting as a `gfx::Size`